server/scripts/standalone/setup-standalone-server.sh

```diff
@@ -94,6 +94,52 @@ function download_zig() {
 }
 
 
+function install_dotnet() {
+	# There are no static download links, they're always generated
+	# https://dotnet.microsoft.com/en-us/download/dotnet/9.0
+
+	if [ "" == "$(which dotnet)" ]; then
+		local os=$(uname)
+		if [ "Darwin" == ${os} ]; then
+			# we can use brew on macos. we pass in "yes" to accept all questions
+			yes | brew install dotnet-sdk@preview
```

Comment:
We could use the `dotnet-install.sh`, but for now using "brew" is good enough (up-to-date).